### PR TITLE
Update contributing guide

### PR DESCRIPTION
**Pull request by lauriejim**
My PR is a: 🚀 New feature

Main update on the: Documentation

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -35,21 +31,16 @@
 
 
 
----
+<!-- ⚠️ Make sure to browse the opened and closed issues before submit your issue. -->
```
Comment (soupette) [inline .github/ISSUE_TEMPLATE.md]: before submitting

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,19 +1,24 @@
-# Contribute to the project
+# Contribute to Strapi
 
-Strapi is an open-source project administered by [the Strapi team](http://strapi.io).
+👍🎉 First off, thanks for taking the time to contribute! 🎉👍
```
Comment (soupette) [inline CONTRIBUTING.md]: First off all

Comment (derrickmehaffy) [reply CONTRIBUTING.md]: Well, "first off" works or "first of all".

First off all doesn't @soupette :)

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -35,21 +31,16 @@
...
-*FEATURE REQUEST TEMPLATE*  <!-- Delete this header from your issue. -->
 
-**What is the expected behavior?**
 
+<!-- FEATURE REQUEST TEMPLATE -->
 
+<!-- ⚠️ If you do not respect this template your issue will be closed. -->
 
----
+**What is the expected behavior?**
```
Comment (abdonrd) [inline .github/ISSUE_TEMPLATE.md]: Maybe you prefer a title:

```
### What is the expected behavior?
```

### What is the expected behavior?

---

Example at https://raw.githubusercontent.com/QISKit/qiskit-js/master/.github/ISSUE_TEMPLATE/BUG_REPORT.md

**Pull request merged**